.gitignore

```diff
@@ -134,4 +134,6 @@ keys_1
 keys_local_beacon
 genesis
 node_operator
+node_operator_2
```

Comment:
can just use `node_operator*`

Reply:
might want to test, but i feel there should be something better here

Reply:
this will update in mainnet release